source/topologies/rs-to-sharded.txt

```diff
@@ -0,0 +1,115 @@
+.. _c2c-rs-to-sharded:
+
+=======================================
+Sync a Replica Set to a Sharded Cluster
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+This page outlines the procedure to synchronize data from a replica set to a 
+sharded cluster using :program:`mongosync`. 
+
+.. note::
+
+   ``mongosync`` doesn't support sync from a sharded cluster to a replica set.
+
+To perform a sync between two replica sets, see :ref:`c2c-quickstart`.
+
+Before You Begin
+----------------
+
+Before you begin your sync:
```

Comment:
I think you can omit this blub, because of the ByB header. Also is it possible to add links to each of the bullets such as Define your :ref:`source replica set <my_link>`
```suggestion

```

Reply:
I removed the blurb and added links!